docroot/modules/custom/va_gov_migrate/src/Plugin/migrate/process/VAFieldOfficeHours.php

```diff
@@ -96,6 +101,27 @@ public function multiple() {
 
 }
 
+/**
+ * Normalizes the hours for further parsing.
+ *
+ * @param string $hour_range
+ *   The string of opening and closing hours for a given day.
+ *
+ * @return string
+ *   A string with extraneous characters and letters capitalized.
+ */
+function normalize_hours($hour_range) {
+  // Set the replacement array with all characters to remove or make uppercase.
+  $replace = [
+    "/a\.?m\.?/i" => "AM",
+    "/p\.?m\.?/i" => "PM",
+    "/\–/"   => "-",
+    "/to/"   => "-",
```

Comment:
```suggestion
    "/\–/" => "-",
    "/to/" => "-",
```
I'm not sure how this is not getting codesniffer errors when you try to commit.  phpcs should be catching this.